.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "Tag (Optional, without 'v' e.g. 1.2.3)"
```

Comment:
is `required: true` the default?

Reply:
I think `required: false` is the default. But in any case, I will make sure to specify it.